jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/ProfileToolkitCredentialsProviderFactoryTest.kt

```diff
@@ -396,10 +404,7 @@ class ProfileToolkitCredentialsProviderFactoryTest {
                 assertThat(sessionCredentials.sessionToken()).isEqualTo("FooSessionToken")
             }
 
-        // Mac timestamp is 1 sec granularity
-        Thread.sleep(1000)
```

Comment:
Cool we can finally remove this sleep

Reply:
Grand total these tests run about 30s faster on Mac